### DISCUSSION TITLE
Support multiple calendar configs

### DISCUSSION THREAD

**Discussion by glennjacobs** [saade/filament-fullcalendar#126] [Ideas]
While using this plugin to create a resource timeline, I noticed that all the configuration is at the plugin level. So it seems that if I wanted two instances of a calendar with different configs, it's essentially impossible currently.

Maybe it would be good if the config could also be set at the widget level?

**Comment by sitenzo** [comment]
@glennjacobs 
I was just reading this because i looked for something just like this.
Did not see any Pull request so i made one [#130](https://github.com/saade/filament-fullcalendar/pull/130)

@saade 
Can you check my code and if you are happy merge it?
